client/state/selectors/get-current-user-localized-payment-methods.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { generateLangCcLocale } from 'state/i18n/util';
+import { getCurrentUserLocale } from 'state/current-user/selectors';
+import { getGeoCountryShort } from 'state/geo/selectors';
+
+/**
+ * Returns the preferred payment methods for the current locale.
+ *
+ * @param  {Object}  state       Global state tree
+ * @return {Array}               Preferred payment methods
+ */
+export function getCurrentUserLocalizedPaymentMethods( state ) {
```

Comment:
This sounds a bit like we're translating payment methods rather than choosing them. What about `getPaymentMethodsForCurrentUserLocale`? Or even go the other way with `getCurrentUserPaymentMethods`? (I can't think of a reason for consumers to care why we're picking the payment methods we are)